src/core/currency/wallet/currency-wallet-api.ts

```diff
@@ -201,9 +201,9 @@ export function makeCurrencyWalletApi(
       const accountState = input.props.state.accounts[accountId]
       const allTokens = accountState.allTokens[pluginId] ?? {}
 
-      const enabledTokenIds = tokenIds.filter(
-        tokenId => allTokens[tokenId] != null
-      )
+      const enabledTokenIds = tokenIds
+        .filter(tokenId => allTokens[tokenId] != null)
+        .filter((v, i, a) => a.findIndex(v2 => v2 === v) === i)
```

Comment:
This is weirdly obfuscated, like it came from Stack Overflow or something. Can we clean it up? The core has an existing `uniqueStrings` helper function that might work.

Reply:
Came from the GUI, actually. Will replace with `uniqueStrings`